frontend/src/app/header/header.component.html

```diff
@@ -171,7 +156,17 @@
         <mat-icon>menu</mat-icon>
       </button>
     </mat-toolbar-row>
+    <mat-toolbar-row class="search">
+      <p>A centralized system for autism-focused research, resources and supports for families and professionals.</p>
```

Comment:
This text doesn't exactly match their request: A centralized system for autism research, resources and supports for individuals, families and professionals.

Reply:
I put the text Eleanor added in.